Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -1522,56 +1522,56 @@ void vtkMRMLMarkupsNode::ConvertOrientationWXYZToMatrix(double orientationWXYZ[4
     return;
     }
 
-  // convert to radians
-  angle = vtkMath::RadiansFromDegrees(angle);
+// convert to radians
+angle = vtkMath::RadiansFromDegrees(angle);
 
-  // make a normalized quaternion
-  double w = cos(0.5*angle);
-  double f = sin(0.5*angle) / sqrt(x*x + y * y + z * z);
-  x *= f;
-  y *= f;
-  z *= f;
+// make a normalized quaternion
+double w = cos(0.5 * angle);
+double f = sin(0.5 * angle) / sqrt(x * x + y * y + z * z);
+x *= f;
+y *= f;
+z *= f;
 
-  // convert the quaternion to a matrix
-  double matrix[4][4];
-  vtkMatrix4x4::Identity(*matrix);
+// convert the quaternion to a matrix
+double matrix[4][4];
+vtkMatrix4x4::Identity(*matrix);
 
-  double ww = w * w;
-  double wx = w * x;
-  double wy = w * y;
-  double wz = w * z;
+double ww = w * w;
+double wx = w * x;
+double wy = w * y;
+double wz = w * z;
 
-  double xx = x * x;
-  double yy = y * y;
-  double zz = z * z;
+double xx = x * x;
+double yy = y * y;
+double zz = z * z;
 
-  double xy = x * y;
-  double xz = x * z;
-  double yz = y * z;
+double xy = x * y;
+double xz = x * z;
+double yz = y * z;
 
-  double s = ww - xx - yy - zz;
+double s = ww - xx - yy - zz;
 
-  orientationMatrix[0] = xx * 2 + s;    // (0,0)
-  orientationMatrix[3] = (xy + wz) * 2; // (1,0)
-  orientationMatrix[6] = (xz - wy) * 2; // (2,0)
+orientationMatrix[0] = xx * 2 + s;    // (0,0)
+orientationMatrix[3] = (xy + wz) * 2; // (1,0)
+orientationMatrix[6] = (xz - wy) * 2; // (2,0)
 
-  orientationMatrix[1] = (xy - wz) * 2; // (0,1)
-  orientationMatrix[4] = yy * 2 + s;    // (1,1)
-  orientationMatrix[7] = (yz + wx) * 2; // (2,1)
+orientationMatrix[1] = (xy - wz) * 2; // (0,1)
+orientationMatrix[4] = yy * 2 + s;    // (1,1)
+orientationMatrix[7] = (yz + wx) * 2; // (2,1)
 
-  orientationMatrix[2] = (xz + wy) * 2; // (0,2)
-  orientationMatrix[5] = (yz - wx) * 2; // (1,2)
-  orientationMatrix[8] = zz * 2 + s;    // (2,2)
+orientationMatrix[2] = (xz + wy) * 2; // (0,2)
+orientationMatrix[5] = (yz - wx) * 2; // (1,2)
+orientationMatrix[8] = zz * 2 + s;    // (2,2)
 }
 
 //----------------------------------------------------------------------
 vtkPoints* vtkMRMLMarkupsNode::GetCurvePoints()
 {
   this->CurveGenerator->Update();
   if (!this->CurveGenerator->GetOutput())
-    {
+  {
```

Comment:
we should still use old-style brace indentation (but hopefully we can get rid of it soon)